src/Cli/Commands/InitOptions.cs

```diff
@@ -34,6 +34,7 @@ public class InitOptions : Options
             bool restDisabled = false,
             string graphQLPath = GraphQLRuntimeOptions.DEFAULT_PATH,
             bool graphqlDisabled = false,
+            CliBoolean restRequestBodyStrict = CliBoolean.True,
```

Comment:
Can you clarify whether this `CliBoolean.True` is used even for cosmos?

Reply:
I'm unclear what the impact is of have this be true verus none.

Reply:
Good catch, we don't even need to provide a default value here to the param. Will update this.